src/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ function(valhalla_module)
   target_compile_definitions(${library}
     PUBLIC
       RAPIDJSON_HAS_STDSTRING
+      HAS_REMOTE_API=0
```

Comment:
this disables the howard hinnat library from trying to fetch timezone database files over http with curl